h/views/activity.py

```diff
@@ -88,17 +90,22 @@ def user_link(userid):
         }
 
 
-@view_defaults(route_name='activity.group_search',
-               renderer='h:templates/activity/search.html.jinja2')
+@view_defaults(route_name='group_read',
+               renderer='h:templates/activity/search.html.jinja2',
+               effective_principals=security.Authenticated,
+               has_feature_flag='search_page',
+               has_permission='read')
```

Comment:
`feature_flag` might be a better name than `has_feature_flag`, as it matches better with the naming style of Pyramid's built-in predicates. On the other hand currently all of our custom predicates are named `has_*` and it's nice to distinguish custom from built-in predicates. (Also `has_permission` can't be `permission` because there's already a built in, non-predicate argument with that name.) So I'm not sure. The names can always be improved later!